CHANGELOG.md

```diff
@@ -1,31 +1,34 @@
 
 # Unreleased
 
-* add CHANGELOG.md. [PR#36](https://github.com/catalystneuro/fee-lab-to-nwb/pull/36)
-* Create PULL_REQUEST_TEMPLATE.md [`0d26f9d`](https://github.com/catalystneuro/fee-lab-to-nwb/commit/0d26f9d44087c755f0a42b9018e45e36c915cef2)
-* add first entry to upcoming changes [`a13ce52`](https://github.com/catalystneuro/fee-lab-to-nwb/commit/a13ce52d67116ad7c59d6fc1294e317c67e1e604)
```

Comment:
Merged Commits that don't have corresponding PR's show up when also having this in the template:

```
  {{#each commits}}
    * ...
  {{/each}}
```